packages/firestore/test/unit/local/indexeddb_persistence.test.ts

```diff
@@ -1144,7 +1144,7 @@ describe('IndexedDb: allowTabSynchronization', () => {
       'clientA',
       /* multiClient= */ false,
       async db => {
-        db.injectFailures = true;
+        db.injectFailures = { updateClientMetadataAndTryBecomePrimary: true };
```

Comment:
I think these tests are now broken in `master` because I merged two conflicting PRs that were both passing all tests on their own.